pallets/subtensor/src/uids.rs

```diff
@@ -9,6 +9,26 @@ impl<T: Config> Pallet<T> {
         SubnetworkN::<T>::get(netuid)
     }
 
+    fn clear_element_at<N>(position: u16) -> impl Fn(&mut Vec<N>)
```

Comment:
We are almost there! can you please add doc comments to this , as it would break some lints down the line. 

I will get this approved today , and move it through our deployment process. I estimate this change would be on testnet in about 2 weeks

Reply:
right, added.

thanks!

Reply:
@distributedstatemachine could you take a look at this again?